src/platforms/common/data-management/debug-files.mdx

```diff
@@ -304,6 +334,19 @@ files are uploaded in the same invocation of the upload command. Otherwise, the
 identifiers diverge and Sentry might not be able to resolve the correct file
 for symbolication.
 
+</PlatformSection>
+
+### WASM Build IDs
+
+WebAssembly does not yet support build IDs.  The option proposed to
+implement build IDs for WebAssembly ([Build ID Section for WASM](https://github.com/WebAssembly/tool-conventions/issues/133))has not yet found widespread adoption.  Instead, we use a custom
+extension to WebAssembly.
+
+Our recommendation is to embed a UUID in the `build_id` custom section as
+raw binary.  Our `wasm-split` tool can do this for you automatically.
```

Comment:
_nit_: Add another link to `wasm-split` so that readers do not have to refer to the other section of the page.